analyzers/tests/SonarAnalyzer.Test/Helpers/SyntaxHelperTest.cs

```diff
@@ -152,7 +152,7 @@ public void NameIsOrNames_CS(bool expected, params string[] orNames)
         [DataTestMethod]
         [DataRow("Strasse", "Straße")]
         [DataRow("\u00F6", "\u006F\u0308")] // 00F6 = ö; 006F = o; 0308 = https://www.fileformat.info/info/unicode/char/0308/index.htm
-        public void NameIsCulture_CS(string identifierName, string actual)
+        public void NameIs_StringComparisonOrdinal_ReturnsFalse(string identifierName, string actual)
```

Comment:
Nitpick, I would remove `StringComparisonOrdinal` since it provides too much detail about the implementation. If the implementation changes the test name would have to change again. 

I would prefer to give more details about the tested scenario instead - different characters -> different results.

On top of this, we might want to add a test that compares `\u00F6` with `\u00F6` and returns true.